lib/is-valid-move.js

```diff
@@ -26,6 +26,10 @@ function isValidMove(gameState, proposedMove) {
         return false;
     }
     
+    if (!activePlayer.barPieces && proposedMove.isBar === true) {
```

Comment:
Do you really need the `=== true`? Can `proposedMove.isBar` be any truthy value other than `true`?